README.md

```diff
@@ -15,30 +20,35 @@ For DevOps and SRE teams, this charm will make operating Jenkins simple and stra
 through Juju's clean interface. Allowing both kubernetes and machine agent relations, it supports
 multiple environments for automation.
 
+For information about how to deploy, integrate, and manage this charm, see the Official [jenkins-k8s charm Documentation](https://charmhub.io/jenkins-k8s/docs).
 
-## Project and community
+## Get started
+
+To begin, refer to the [Getting Started](https://charmhub.io/jenkins-k8s/docs/tutorial-getting-started) tutorial for step-by-step instructions.
+
+### Basic operations
+
+#### Expose jenkins-k8s through ingress
+
+See the [Expose jenkins-k8s through ingress](https://charmhub.io/jenkins-k8s/docs/tutorial-getting-started#expose-jenkins-k8s-through-ingress) section in the jenkins-k8s-operator documentation.
 
-The Jenkins-k8s Operator is a member of the Ubuntu family. It's an open source
-project that warmly welcomes community projects, contributions, suggestions,
-fixes and constructive feedback.
-* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
-* [Get support](https://discourse.charmhub.io/)
-* [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
-* [Contribute](https://charmhub.io/jenkins-k8s/docs/contributing)
-* [Getting Started](https://charmhub.io/jenkins-k8s/docs/tutorial-getting-started)
-Thinking about using the Jenkins-k8s Operator for your next project? 
-[Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
+#### Integrate with the jenkins-agent and the jenkins-agent-k8s charm
 
----
+See the [deploy and integrate k8s agents](https://charmhub.io/jenkins-k8s/docs/tutorial-getting-started#deploy-and-integrate-k8s-agents) section and the [deploy and integrate machine agents](https://charmhub.io/jenkins-k8s/docs/tutorial-getting-started#deploy-and-integrate-machine-agents-optional) section in the jenkins-k8s-operator documentation.
 
-For further details,
-[see the charm's detailed documentation](https://charmhub.io/jenkins-k8s/docs).
+#### Use agent-discovery-ingress integration to integrate with "external" agents
+
+See the [how-to-integrate-with-external-agents](https://charmhub.io/jenkins-k8s/docs/how-to-integrate-with-external-agents) section in the jenkins-k8s-operator documentation.
+
+## Learn more
+
+- [Read more](https://charmhub.io/jenkins-k8s/docs)
+
```

Comment:
- Is there an official website for Jenkins that is worth listing here?
- Similarly, are there any developer docs for Jenkins?
- Finally, could you add an item `[Troubleshooting](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)` to this list?